src/Script/ticketController.js

```diff
@@ -1,4 +1,4 @@
-$(document).ready(() => { loadSummary(); loadProgress(); })
+$(document).ready(() => { loadSummary(); loadProgress(); NavbarActivater.projectActiveTab(); })
```

Comment:
This Script file will be made redundant, make a new ticket.js file in the public folder and add it is, remember, all JS files will be removed in the future and the new ones will replace it

Reply:
fixed